cmd/Rammus.GG/main.go

```diff
@@ -1,7 +1,63 @@
 package main
 
-import "fmt"
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/go-chi/chi/v5/middleware"
+)
 
 func main() {
-	fmt.Println("hello, world!")
+	r := chi.NewRouter()
+
+	r.Use(middleware.Logger)
+
+	r.Get("/", func(w http.ResponseWriter, r *http.Request) {
+		_, err := w.Write([]byte("OK"))
```

Comment:
Error handling do `ResponseWriter.Write` é um tanto tricky. [Essa resposta](https://stackoverflow.com/a/43976633) do SO dá uma boa luz em como lidar com isso.

Dá para usar o _[io.WriteString](https://pkg.go.dev/io#WriteString)_ ou _[fmt.Fprint](https://pkg.go.dev/fmt#Fprint)_ (`Fprintf` e variações...), sempre que tivermos um _[io.Writer](https://pkg.go.dev/io#Writer)_, no caso de escrevermos `string`.


```suggestion
		_, err := io.WriteString(s, "OK")
		// ou
		_, err := fmt.Fprintf(s, "OK")
```

Se depois formos usar JSON, a situação muda um pouco.

Nota:
_[ResponseWriter.Write](https://pkg.go.dev/net/http#ResponseWriter.Write)_ tem muitas nuancias, vale ler a doc.